SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -240,4 +242,9 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 #if !UE_BUILD_SHIPPING
 	int32 ConsiderListSize = 0;
 #endif
+
+#if WITH_EDITOR
+	static const int32 EDITOR_TOMBSTONED_ENTITY_TRACKING_RESERVATION_COUNT = 256;
```

Comment:
Is this just to prevent more allocation at runtime?

Reply:
Yep, just pre-reserving some space in the array to avoid the realloc hits.